.github/workflows/lint-and-test.yml

```diff
@@ -49,6 +49,6 @@ jobs:
         run: pnpm --filter ...[${{ steps.since.outputs.SINCE }}] type-check
 
       - name: Test
-        run: pnpm --filter ...[${{ steps.since.outputs.SINCE }}] test:ci
+        run: VCR_MODE=playback pnpm --filter ...[${{ steps.since.outputs.SINCE }}] test:ci
```

Comment:
I know we have different opinions on this but it should only be playback when `HF_ACCESS_TOKEN` is not defined